app/tests/spec/views/mixins/signup-mixin.js

```diff
@@ -222,16 +165,6 @@ define(function (require, exports, module) {
           assert.equal(args[0], 'afterSignUp');
           assert.deepEqual(args[1], account);
         });
-
-        it('calls view.navigate correctly', function () {
```

Comment:
The mixin no longer handles navigating after `afterSignUp`, the broker's default behavior is to navigate to `confirm`.